### ISSUE TITLE
Internal Server Error on /counter page

### ISSUE THREAD

**Issue by bishopZ** [BausCode/Baus-Boilerplate#16]
You all have done a great job here. This is the best React-based boilerplate I have seen. One problem though.

When I run this repo and go to the home page, I can click on the Counter nav item- the URL changes to /counter and the counter page is displayed. If I refresh the page, I get an "Internal Server Error".

In development I get this error
```
Invariant Violation: Could not find "store" in either the context or props of "Connect(Counter)". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(Counter)".
   at invariant (/git/Baus-Boilerplate/node_modules/react-redux/node_modules/invariant/invariant.js:42:15)
   at new Connect (/git/Baus-Boilerplate/node_modules/react-redux/lib/components/connect.js:131:36)
   at [object Object].ReactCompositeComponentMixin._constructComponentWithoutOwner (/git/Baus-Boilerplate/node_modules/react/lib/ReactCompositeComponent.js:280:27)
```

My guess is that the file app.js does not run when hitting a subpage route.

Any help appreciated.

**Comment by bishopZ** [comment]
You can see it happen here https://multipage-quiz.herokuapp.com/counter

**Comment by bishopZ** [comment]
I tracked the issue down to the server-side rendering in server.js. The issue was introduced in [this commit](https://github.com/BausCode/Baus-Boilerplate/commit/f7b3cb7f2f137faf3959b21ff96e4cdcdb9e06d2).

I resolved the issue on my server by changing this
```
let html = renderToString(<RouterContext {...renderProps} />);
res.status(200).render('main', {env: env, content: html});          
```
to this.
```
res.status(200).render('main', {env: env});
```

Doing this removes the server-side rendering, but allows the Routes to work as intended and enables me to continue moving forward. Let's chat about a longer-term solution when you have a chance.
